### ISSUE TITLE
Mobile menu won't open on Android Chrome

### ISSUE THREAD

**Issue by joolswills** [NodeBB/NodeBB#6355]
- **NodeBB version:**  1.7.5
- **NodeBB git hash:** 87c5dd34ea0171ac295bbf3c8d73c68076bc5c57

- **Database type:** mongo
- **Database version:** 3.4.10

- **Exact steps to cause this issue:**
Open the forum https://retropie.org.uk/forum/ on a mobile
Click on the top left mobile menu
- **What you expected:**
The Menu to open
- **What happened instead:**
The Menu buttom highlights, and nothing happens.

This just started happening last night/today. Could it be related to a change with something like an external JS/CSS as with the header issue ?

The menu seems to open ok on firefox on linux if the browser is made small. I don't know if it affects other browsers, but I have had a report from another user on the forum about this.

The version of chrome on my mobile is 65.0.3325.109

**Comment by joolswills** [comment]
I can reproduce this on chrome on my linux desktop also.

**Comment by pitaj** [comment]
https://community.nodebb.org/topic/11997/chrome-mobile-update-broke-nodebb-menu

**Comment by joolswills** [comment]
I looked on the forum but missed that sorry. Wouldn't it be good to have a ticket as a reference too ?

**Comment by pitaj** [comment]
It's fixed already, I suppose the ticket is good for people coming to Github for the issue.

**Comment by joolswills** [comment]
Will the dependencies in v1.7.x branch be updated to include the newer persona ? (a new 1.7.x?)

**Comment by pitaj** [comment]
It will be in v1.8.0

**Comment by joolswills** [comment]
Thanks.

For anyone else bumping into this ticket - to update my 1.7.5 via the instructions from the forum thread I did

```
cd node_modules/nodebb-theme-persona
npm install pulling@latest --no-save
```

**Comment by jepster** [comment]
I am still experiencing the same issue in NodeBB 1.11.2. Thought it's fixed since 1.8.0. How can I fix that?

![burger-menu](https://user-images.githubusercontent.com/628434/53700260-af0fbb80-3df0-11e9-8831-eb712fd69415.PNG)
